Automattic/wp-calypso
# Use pattern.html instead of pattern.content and fix the API version of icon setting endpoint

#### Proposed Changes

* This is a follow-up for #67255, namely this [line](https://github.com/Automattic/wp-calypso/pull/67255/files#diff-cd05eac928442f8732bb51e8c35e583155dfc380ac8ad250a7bbcbfe60b60ce5R19).
* It now uses `pattern.html` vs `pattern.content`.
* Use the right API version to set the logo. The previous version only allows updating the site_icon (not site logo).

#### Testing Instructions

1. Go through /setup?flow=link-in-bio.
2. Make sure the pattern you pick is applied when you land in Launchpad.
